libraries/SITL/SIM_Aircraft.cpp

```diff
@@ -75,6 +75,7 @@ Aircraft::Aircraft(const char *frame_str) :
     enum ap_var_type ptype;
     ahrs_orientation = (AP_Int8 *)AP_Param::find("AHRS_ORIENTATION", &ptype);
 
+    // ahrs_orientation->get() returns ROTATION_NONE here, regardless of the actual value
```

Comment:
Why is this?

Reply:
AHRS_ORIENTATION = 39 =  Pitch315 should work without the need for custom, not that allowing custom isn't a good idea anyway

Reply:
I suppose it might be something to do with the timing of the constructor being call.
Huh, I found custom before I found Pitch315... my bad luck :)

Reply:
odd that fetching AHRS_CUSTOM_ROLL ect works when AHRS_ORIENTATION does not

Reply:
that doesn't work in the constructor either

Reply:
load_parameters() is called after the constructor for SIM_Aircraft, so I don't see how any calls into AP_Param would work...